Common/src/main/resources/defaultresources/excavated_variants/excavated_variants/variants/deepresonance.json5

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (C) 2023 Luke Bemish and contributors
+ * SPDX-License-Identifier: LGPL-3.0-or-later
+*/
+
+{
+  mod_id: 'deepresonance',
+  provided_stones: [],
+  provided_ores: [
+	{
+      id: 'resonating_ore',
+      lang: {
+        en_us: 'Resonating Ore',
+      },
+      block_id: [
+        'deepresonance:resonating_ore_stone',
+        'deepresonance:resonating_ore_deepslate',
+      ],
+      types: [
+        'stone',
+      ],
+      stone: [
+        'stone',
+        'deepslate',
+      ],
+	},
+	{
+      id: 'nether_resonating_ore',
+      orename: 'resonating_ore',
+      lang: {
+        en_us: 'Resonating Ore',
+      },
+      block_id: [
+        'deepresonance:resonating_ore_nether',
+      ],
+      types: [
+        'nether',
+      ],
+      stone: [
+        'netherrack',
+      ],
+	},
+	{
+      id: 'end_resonating_ore',
+      orename: 'resonating_ore',
+      lang: {
+        en_us: 'Resonating Ore',
+      },
+      block_id: [
+        'deepresonance:resonating_ore_end',
+      ],
+      types: [
+        'end',
+      ],
+      stone: [
+        'end_stone',
+      ],
+	},
+  ],
+}
```

Comment:
Can we make sure files have trailing newlines?

Reply:
What do you mean? Empty spaces at the end?

I guess I could, but I'm not sure why it should be there.

Reply:
Eh, you're right, looks like I didn't fix that issue till a newer version, so no need. Just fix the other stuff and it's good

Reply:
Already did.